### PR TITLE
(PCP-374) Update Catch configuration

### PR DESCRIPTION
**Pull request by parisiale**
Updating tests/main.cc to Catch v1.4.0 changes.

### CODE REVIEW

**Comment by parisiale** [comment]
Don't merge; a new leatherman release is required.

**Comment by parisiale** [comment]
This can now be merged.

File: lib/tests/main.cc [hunk 1]
```diff
@@ -12,23 +16,28 @@
...
+int main(int argc, const char** argv) {
 #ifdef ENABLE_LOGGING
+    leatherman::logging::setup_logging(boost::nowide::cout);
     leatherman::logging::set_level(leatherman::logging::log_level::debug);
 #endif
 
-    // configure the Catch session and start it
-    Catch::Session test_session;
+    // Create the Catch session, pass CL args, and start it
+    Catch::Session test_session {};
```
Comment (mruzicka) [inline lib/tests/main.cc]: What's the difference between:
```C++
Catch::Session test_session;
```
and
```C++
Catch::Session test_session {};
```
isn't the constructor called in the first case or what?

Comment (parisiale) [inline lib/tests/main.cc]: The latter is more consistent with the rest of the code base.

Comment (mruzicka) [inline lib/tests/main.cc]: I see

**Pull request merged**